src/order/order-request-body.ts

```diff
@@ -1,4 +1,4 @@
-import { CreditCardInstrument, VaultedInstrument } from '../payment';
+import { PaymentInstrument } from '../payment/payment';
```

Comment:
Prefer to import from index if importing files from a different root folder.

Reply:
👍